app/frontend/src/authConfig.ts

```diff
@@ -0,0 +1,70 @@
+import { AuthenticationResult, IPublicClientApplication } from "@azure/msal-browser";
+
+interface AuthSetup {
```

Comment:
Was this pasted from somehere? (Given all the comments) It's fine if so, but may be useful to note.

Reply:
yes, it was pasted from an existing sample, it's possible the attribution got wiped here

Reply:
at this point it's been heavily refactored (auth setup shared from the server), but the original sample is really useful so i added a lnk